public/_redirects

```diff
@@ -5,7 +5,6 @@ https://asyncapi.com/* https://www.asyncapi.com/:splat 301!
 # Docs
 /docs /docs/getting-started 301!
 /docs/tooling /docs/community/tooling 301!
-/docs/specifications/latest /docs/specifications/2.0.0 302!
```

Comment:
we should keep it, and it should be updated by the action on the spec side, feel free to wrap this line in a kind of comment, so it is easier to parse + you have clear info for maintainers to not modify it manually.

have a look at https://raw.githubusercontent.com/asyncapi/generator/master/README.md and the marker `<!-- TEMPLATES-LIST:START -->` as an example, handled by this code https://github.com/asyncapi/generator/blob/master/.github/templates-list-validator/index.js#L29